wiki/Manual.md

```diff
@@ -2423,32 +2423,9 @@ If `since` is not specified the `fetchOHLCV` method will return the time range a
 
 ### Notes On Latency
 
-Trading strategies require fresh up-to-date information for technical analysis, indicators and signals. Building a speculative trading strategy based on the OHLCV candles received from the exchange may have critical drawbacks. Developers should account for the details explained in this section to build successful bots.
```

Comment:
@ttodua Actually I think the original text sounds more professional, why are we changing it? Can't we just update the buildOHLCV part?

Reply:
I don't think those big paragraphs actually add any benefit (other than disturbing users for reading large texts). when we have heading about "Notes on latency" we should describe what's we are about, and not some generic advises to users, which for sure already know that "Trading strategies require fresh up-to-date information for technical analysis" and "latency is important" and etcetera. my proposed version is half of the text with all those needed info. I could rewrite it or insert back the part of the original text too, but if you prefer the original version completely, oky, no problem, i'll close this ✌️

Reply:
also original version has some broken reference links too